From 30ec4912d5c51510571d93e899d3a424a32c4950 Mon Sep 17 00:00:00 2001
From: aken_liu <aken_liu@accton.com.tw>
Date: Wed, 21 Nov 2018 13:12:52 +0800
Subject: [PATCH] workaround for hard-reset in R0A blade check the blade
 version and then execute the reset function.  Hard reset: R0A: execute warm
 reset. R0B: execute hard reset
 UPDATED for T650:  This dummy patch is used to keep patches somewhat in
 sync with T600.  Keep line numbers aligned for later patches.

---
 drivers/hwmon/accton_t600_cpld.c | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/drivers/hwmon/accton_t600_cpld.c b/drivers/hwmon/accton_t600_cpld.c
index bac97d5..c636f63 100644
--- a/drivers/hwmon/accton_t600_cpld.c
+++ b/drivers/hwmon/accton_t600_cpld.c
@@ -301,6 +301,21 @@ static ssize_t reset(struct device *dev, struct device_attribute *da,
         return -EINVAL;
     }
 
+    /* Dummy patch.  Keep line numbers aligned with T600 version */
+    /*
+     *
+     *
+     *
+     *
+     *
+     *
+     *
+     *
+     *
+     *
+     *
+     */
+
     mutex_lock(&data->update_lock);
     if (reset == 1) {
         /* RESET_ALL when SW hard reset */
-- 
1.9.1

